Uppsala/UppEt3.xml

```diff
@@ -0,0 +1,557 @@
+                                    that one <foreign xml:lang="gez-tr">qamar</foreign> equals 532 years. Thirteen 
+                                    <foreign xml:lang="gez-tr">qamar</foreign> thus equal 6916 years, to which (according to the text)
+                                    one adds 84 "Years of the Evangelists"
+                                    (<foreign xml:lang="gez">ወትዌስክ፡ ዲቤሁ፡ ዓመተ፡ ወንጌ<sic resp="JK">ለ</sic>ዊያን፡ 
+                                        <hi rend="rubric">፹፬፡</hi></foreign>),
+                                    resulting in a sum of 7000 years.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i2.2">
+                                <locus from="57v" to="60r"/>
+                                <title type="complete">About the Jubilees (<foreign xml:lang="gez">ኢዮቤልዩ፡</foreign>, 
+                                    <foreign xml:lang="hebr">יובל</foreign>)</title>
+                                <note><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">29</citedRange></bibl> notes
+                                    that one <foreign xml:lang="gez-tr">ʾiyobelyu</foreign> equals 49 years. 142 
+                                    <foreign xml:lang="gez-tr">ʾiyobelyu</foreign> thus equal 6958 years, to which (according to the text)
+                                    one adds 6 <foreign xml:lang="gez-tr">subāʿe</foreign>
+                                    (<foreign xml:lang="gez">ወ<hi rend="rubric">፮</hi>ሱባዔ፡</foreign>),
+                                    resulting in a sum of 7000 years. As noted by 
+                                    <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">29</citedRange></bibl>,
+                                    the section also contains a section with "Correspondances between the <foreign xml:lang="gez-tr">qamar</foreign> 
+                                    and the years" (<foreign xml:lang="gez">ዕሪናሁ፡ ለቀመር፡ ወለዓመታት፡</foreign>).</note>
+                            </msItem>
+                            <msItem xml:id="ms_i2.3">
+                                <locus from="60r" to="60v"/>
+                                <title type="complete">Chronology of Enoch</title>
+                                <note><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">29</citedRange></bibl> notes
+                                    that it consists of seven <foreign xml:lang="gez-tr">sanābət</foreign> of 700 years each,
+                                    resulting in a sum of 7000 years.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i2.4">
+                                <locus from="60v" to="61r"/>
+                                <title type="complete">Chronology of the Sibyl (<foreign xml:lang="gez">ሰቤላ፡</foreign>)</title>
```

Comment:
The ID of "Chronology of the Sybil" is LIT4992HassabSabela. If all mss which Löfgren indicates indeed contain the same work, the ID will hopefully be used in the descriptions. I would find them and see if a kind of harmonisation is needed or not (If BNFet64 contains the same text, the ID may probably be the same; but in BNFet64 it is LIT3851Hassab, here above is the general record LIT1195Bahrah).  I would also strongly recomment to quote a little bit of text for each of the part texts, this will be very helpful (It would be nice if you could quote from the incipit under LIT4992HassabSabela)

Reply:
Thank you! I can only see one manuscript with LIT4992HassabSabela, and there I only have the incipit in Turaev, which is not exactly the same as in the Uppsala manuscript. Can we be sure that they are the same text?

I looked at the text in BNFet64, and it has a lot of similarities with the text in the Uppsala manuscript, but the elements don't come in the same order and I'm not sure that it has exactly the same contents. I don't know if it's better to say that they are both LIT3851Hassab or to use the more general ID. What do you say?

I can add incipits!

Reply:
Of course we cannot be sure (in my opinion, the chance is high), but we can signal their relationship (maybe Löfgren had a chance to check the text closer!). You can also use `<cert=""/>`

Reply:
To do this interlinking/integrating work with "text units" is the constant background research task done in Bm.  If you are sure that both texts are related but have differences, you can still create another record and interlink both, explaining also your reservations (quoting meaningful chunks of text is always helpful, the lack of quotation is a weakness of old catalogues, like of Turaev).

Reply:
Ok, I add LIT4992HassabSabela. It seems reasonable at least until someone takes a closer look at the manuscripts.